File: cpp/Estruturas condicionais/baskara.cpp
#include <iostream>
#include <cmath>
using namespace std; 
int main() {
    double A, B, C, delta, x1, x2;

    cout << "Qual o valor de A" << endl;
    cin >> A;

    cout << "Qual o valor de B" << endl;
    cin >> B;

    cout << "Qual o valor de C" << endl;
    cin >> C;

    delta = (B * B) - 4 * A * C;
    
    if (delta < 0 || A == 0) {
       cout << "Esta equacao nao possui raizes reais" << endl;
    }
    
    else {
        x1 = (-B + sqrt(delta)) / (2 * A);
        x2= (-B - sqrt(delta)) / (2 * A);
        cout << "As raizes da equacao sao: " << x1 << " e " << x2 << endl;
        }

    


    return 0;
}

File: java/Estruturas condicionais/Baskara.java
import java.util.Locale;
import java.util.Scanner;

public class Baskara {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double a, b, c;
        double delta, x1, x2;

        System.out.print("Digite o valor de a: ");
        a = sc.nextDouble();

        System.out.print("Digite o valor de b: ");
        b = sc.nextDouble();

        System.out.print("Digite o valor de c: ");
        c = sc.nextDouble();

        delta = b * b - 4 * a * c;

        if (delta < 0) {
            System.out.println("Esta equação não possui raízes reais.");
        } else {
            x1 = (-b + Math.sqrt(delta)) / (2 * a);
            x2 = (-b - Math.sqrt(delta)) / (2 * a);

            System.out.printf("X1 = %.4f%n", x1);
            System.out.printf("X2 = %.4f%n", x2);
        }

        sc.close();
    }
}